Assets/LeapMotion/Scripts/HandPool.cs

```diff
@@ -17,53 +17,94 @@ public class HandPool :
     HandFactory {
 
     [SerializeField]
-    private List<IHandModel> ModelCollection;
-    public List<IHandModel> ModelPool;
+    private List<ModelPair> ModelCollection;
+    public List<ModelGroup> ModelPool;
```

Comment:
ModelPool should be made private (or projected) with a SerializeField attribute, since it is not something we want other scripts being able to mess with.